docs/user_guide.md

```diff
@@ -438,30 +484,35 @@ To use tools from toolbox in your agent network, simply call them with field `to
 
 ## Logging and debugging
 
-To debug your code, set up your environment per instructions [here](https://github.com/cognizant-ai-lab/neuro-san-studio).
+To debug your code, set up your environment per these [instructions](https://github.com/cognizant-ai-lab/neuro-san-studio).
 Furthermore, please install the build requirements in your virtual environment via the following commands:
 
-```
+```bash
 . ./venv/bin/activate
 pip install -r requirements-build.txt
 ```
 
-2. Suppose you want to debug the coded tool for `music_nerd_pro` agent network. Add the following lines of code to the `music_nerd_pro`'s coded tool Python file (E.g., to the first line of `invoke` method in `Accountant` [class](https://github.com/cognizant-ai-lab/neuro-san-studio/blob/main/coded_tools/music_nerd_pro/accounting.py)
+* Suppose you want to debug the coded tool for `music_nerd_pro` agent network. Add the following lines of code to the
```

Comment:
It should not start at 2., but it's a numbered list describing steps to follow. I think it's better to keep it numbered instead of using simple bullet points.